hw_diag/diagnostics/add_gateway_txn_diagnostic.py

```diff
@@ -0,0 +1,42 @@
+import json
+import secrets
+
+from hm_pyhelper.diagnostics import Diagnostic
+from hm_pyhelper.constants.shipping import DESTINATION_WALLETS_KEY
+from hm_pyhelper.miner_json_rpc.exceptions import MinerMalformedAddGatewayTxn
+
+from hw_diag.utilities.miner import create_add_gateway_txn
+from hw_diag.utilities.security import get_verified_data
+
+KEY = 'ADD_GATEWAY_TXN'
+FRIENDLY_NAME = "add-gateway-transaction"
```

Comment:
Use this constant from pyhelper: https://github.com/NebraLtd/hm-pyhelper/blob/master/hm_pyhelper/constants/shipping.py#L2

Reply:
@ilyastrodubtsev can you update [hm-dashboard](https://github.com/NebraLtd/hm-dashboard/blob/081077c2a75b9295265ae85101a3c17362db8e0c/app/device/views.py#L614) to reference the same constant: https://github.com/NebraLtd/hm-pyhelper/blob/27c5c2112475bc1a658bd5d98ecae8d2b862454a/hm_pyhelper/constants/shipping.py#L2